kwc-building-listing.html

```diff
@@ -8,27 +10,136 @@
+                --kwc-challenge-card-cover: {
+                    height: 176px;
+                }
+            }
+
+             * {
+                box-sizing: border-box;
+            }
+
+             *[hidden] {
+                display: none;
+            }
+
+             #pager {
+                @apply --layout-vertical;
+            }
+
+             .card-container {
+                @apply --layout-horizontal;
+                @apply --layout-wrap;
+                @apply --layout-center-justified;
+                margin: 24px 0;
+                position: relative;
+            }
+
+             kwc-challenge-card {
+                width: 184px;
+                margin: 0 12px 32px 12px;
+                cursor: pointer;
+            }
+
+             .h-line {
+                height: 1px;
+                background-color: var(--color-porcelain);
+            }
+
+             .loading {
+                @apply --layout-fit;
+                margin: auto;
+            }
+
+             .loading-label {
+                color: var(--color-abbey);
+                font-size: 24px;
+                font-weight: bold;
+                padding-bottom: 24px;
+                text-align: center;
+            }
+
+             paper-spinner-lite {
+                display: block;
+                margin: auto;
+            }
+        </style>
+        <div class="card-container">
+
+            <div class="loading" hidden$="[[!_challengesLoading]]">
+                <div class="loading-label">Loading...</div>
+                <paper-spinner-lite active="[[_challengesLoading]]"></paper-spinner-lite>
+            </div>
+
+            <template is="dom-repeat" items="[[challenges]]" as="challenge">
+                <kwc-challenge-card
+                    label="Challenge Card"
+                    img-url="[[challenge.image]]"
+                    title="[[challenge.title]]"
+                    challenge-id="[[challenge.slug]]"
+                    completed="[[_isCompleted(challenge, completedChallenges)]]"
+                    on-tap="_clickChallenge"
+                ></kwc-challenge-card>
+      
+            </template>
+        </div>
+    </template>
+
+    <script>
+        Polymer({
+
+            is: 'kwc-building-listing',
+
+            properties: {
+                /*
+                * List of challenges to be displayed
+                */
+                challenges: {
+                    type: Object,
+                    value: () => {
+                        return [];
+                    }
+                },
+                /*
+                * List of completed challenge ids
+                */
+                completedChallenges: {
+                    type: Object,
+                    value: () => {
+                        return [];
+                    }
+                },
+                /*
+                * Computed property to decide to show loading spinner or not.
+                */
```

Comment:
Could we update the comment formatting to match the Polymer standard, with some indentation and two asterisks to begin with:
```
/**
 *
 */
```

Reply:
ok